app/src/main/java/org/oppia/app/topic/overview/TopicOverviewFragmentPresenter.kt

```diff
@@ -59,9 +60,9 @@ class TopicOverviewFragmentPresenter @Inject constructor(
     })
   }
 
-  // TODO(#135): Get this topic-id from [TopicFragment].
   private val topicResultLiveData: LiveData<AsyncResult<Topic>> by lazy {
-    topicController.getTopic(TEST_TOPIC_ID_0)
+    topicId = fragment.arguments?.getString(TOPIC_ACTIVITY_TOPIC_ID_ARGUMENT_KEY) ?: ""
```

Comment:
Can we define this in handleCreateView, instead, and use checkNotNull() to throw if it isn't defined? I don't think TopicController should be able to support loading a topic with an empty ID, or at least it probably won't do something that the UI can handle correctly (e.g. I expect nothing to be displayed--it's probably better to crash for us to detect that bug earlier and address it).

Reply:
Resolved